app/code/Magento/CatalogGraphQl/Model/Resolver/Products/DataProvider/Image/Placeholder.php

```diff
@@ -52,8 +61,9 @@ public function getPlaceholder(string $imageType): string
             return $imageAsset->getUrl();
         }
 
-        return $this->assetRepository->getUrl(
-            "Magento_Catalog::images/product/placeholder/{$imageType}.jpg"
+        return $this->assetRepository->getUrlWithParams(
+            "Magento_Catalog::images/product/placeholder/{$imageType}.jpg",
+            $this->theme->getThemeData()
```

Comment:
Could you please use AppEmulation to get correct URL in all cases ?

Reply:
I updated the PR, thanks